src/pages/cos/LoginCos/LoginCos.js

```diff
@@ -1,8 +1,96 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './LoginCos.scss';
+import { useNavigate } from 'react-router-dom';
 
 function LoginCos() {
-  return;
+  const [idValue, setIdValue] = useState('');
+  const [pwValue, setPwValue] = useState('');
+  const [disabled, setDisabled] = useState(true);
+  const [style, setStyle] = useState({
+    backgroundColor: '#B1DFFC',
+    cursor: 'default',
+  });
+
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate('/main-cos');
+  };
+
+  const handleIdInput = event => {
+    setIdValue(event.target.value);
+    checkResult();
+  };
+
+  const handlePwInput = event => {
+    setPwValue(event.target.value);
+    checkResult();
+  };
+
+  const checkId = value => {
+    return value.includes('@') ? true : false;
+  };
+
+  const checkPw = value => {
+    return value.length >= 5 ? true : false;
+  };
```

Comment:
- 기능을 함수로 분리하는 부분은 좋음
- 삼항연산자로 사용할 필요 없이, 조건 자체로 boolean이 될 수 있음
```suggestion
  const checkId = value => {
    return value.includes('@');
  };

  const checkPw = value => {
    return value.length >= 5;
  };
```
- 추가적으로,
```js
const isInputValid = idValue.includes('@') && pwValue.length >= 5
```
이런 식으로 한꺼번에 변수로 만들어 처리할 수도 있음

Reply:
이 변수 하나로 disabled, style state 모두를 대체할 수 있음